portfolio/src/main/webapp/style.css

```diff
@@ -33,4 +33,63 @@ h1 {
 
 .nav-links a:hover {
   background-color: lightgrey;
+}
+
+#image-links a {
+  float: none;
+  text-align: center;
+  padding-right: 20px;
+}
+
+.center {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+/*image gallery styling */
+
+/* grid: Four equal columns that floats next to each other */
+.column {
+  float: left;
+  width: 25%;
+}
+
+/* Style the images inside the grid */
+.column img {
+  opacity: 0.8;
+  cursor: pointer;
+}
+
+.column img:hover {
+  opacity: 1;
+}
+
+.row:after {
+  content: "";
```

Comment:
nit: get the fields ordered alphabetically, that's usually the best practice we have in the codebase

Reply:
will reorder them.